pokemongo_bot/event_manager.py

```diff
@@ -29,7 +29,6 @@ def event_report(self):
         for event, (color, parameters) in self._registered_events.iteritems():
             print '-'*80
             print 'Event: {}'.format(event)
```

Comment:
oh shit, my bad... don't remove those, here it's the event's report... they should be here